data_pipeline/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "Data_pipeline for trending data sentimental analysis"
```

Comment:
1. Data Pipeline NOT Data_pipeline 
2. This name looks like a description, there's description field for that